providers/gov/nih/normalizer.py

```diff
@@ -12,6 +14,21 @@ def filter_nil(obj):
         return obj
```

Comment:
I can't actually comment on the correct line, but could you fix the indentation here?